bin/knapsack_pro_fixed_queue_rspec_queue_consumed_at_least_once_without_ci_build_id

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
```

Comment:
I guess it's fine to remove it. We have tests on the API side to ensure the API supports requests to Queue API without CI build ID. Tests for API are needed to ensure old knapsack_pro gems work correctly and we won't break anything for customers.